R/f7-inputs.R

```diff
@@ -405,44 +422,12 @@ f7ColorPicker <- function(inputId, label, value = "#ff0000", placeholder = NULL,
     )
   )
 
-
-  inputTag <- shiny::tags$input(
-    type = "text",
-    placeholder = placeholder,
-    id = inputId,
-    class = "color-picker-input"
-  )
-
-  wrapperTag <- shiny::tags$div(
-    class = "list no-hairlines-md",
-    shiny::tags$ul(
-      shiny::tags$li(
-        shiny::tags$div(
-          class = "item-content item-input",
-          shiny::tags$div(
-            class = "item-media",
-            shiny::tags$i(
-              class = "icon demo-list-icon",
-              id = paste0(inputId, "-value")
-            )
-          ),
-          shiny::tags$div(
-            class = "item-inner",
-            shiny::tags$div(
-              class = "item-input-wrap",
-              inputTag
-            )
-          )
-        )
-      )
-    )
-  )
-
-  labelTag <- shiny::tags$div(class = "block-title", label)
-  shiny::tagList(
-    shiny::singleton(pickerProps),
-    labelTag,
-    wrapperTag
+  # TO DO: placeholder?
+  buildPickerInput(
+    inputId,
+    label,
+    config,
+    "color-picker-input"
```

Comment:
@DivadNojnarg this colorPicker used a placeholder in the input tag. Worthwhile to include in `buildPickerInput`? Would be nice for a normal picker too

Reply:
I don't recall if the placeholder is ever shown in the UI. If you manage to make it work, yes we can include it.

Reply:
In the normal picker, yes. In the color picker, perhaps if you don't have an initial value. I don't even know if that works without an initial value 🤔

Reply:
There's a test for the value in f7Picker so that would cause an error. The reason is JS side, I think you really need a value (not 100% sure, you can have a look).

Reply:
I think it's fine, works similar as the example. So I can remove that test, and add a placeholder arg to `buildPickerInput` 👍 

<img width="553" alt="image" src="https://github.com/RinteRface/shinyMobile/assets/60436372/2422c4ee-e7b3-4993-90c4-fd4dc804c6f3">
<img width="553" alt="image" src="https://github.com/RinteRface/shinyMobile/assets/60436372/d2a6ecd8-93a9-4938-8c6d-4318752b799c">

Reply:
That being said, for the color picker its quite useless, you just cannot select no color 🤷‍♀️